qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -26,6 +26,10 @@
                 backend_type = passthrough
                 filename_passthrough = /dev/random
         - with_viostor:
+            # For latest windows guests, changing system disk driver between
+            # virtio_blk and virt_scsi will cause BSOD "INACCESSABLE_DEVICE".
+            # Do a workaround here to avoid such scenario.
+            no virtio_scsi
             driver_name = viostor
             device_name = "Red Hat VirtIO SCSI controller"
             drive_format_image1 = ide
```

Comment:
1. I think no virtio_scsi is better than only virtio_blk, as there are other drive formats.
2. Have confirmed and test by myself, changing system disk between ide and virtio_blk/virtio_scsi won't cause any problem.
Removing "drive_format_image1 = ide" means virtio_blk / virtio_scsi has been installed in unattended_install successfully.
3. Sorry, I have not got your meaning at point 3.